### PR TITLE
Add slicerator v0.9.7

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
@nkeim @caspervdw I am adding slicerator (and pims and trackpy to follow) to conda-forge for linux, mac and windows. This will allow users to easily install this with [conda] (http://conda.pydata.org/docs/) (conda install -c conda-forge keyring). The maintenance burden here is light as all testing and releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/slicerator-feedstock shortly after this PR is finished and merged. If you would like to be added as a maintainer (to make new releases for instance), I can happily add you to the maintainers list. If not, that is totally fine too!  Is there anyone else that you think would want to be pinged about maintaining this build?

File: recipes/slicerator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    requires:
+        - nose
+        - six
+        - numpy
+    commands:
+        - python {{ environ.SRC_DIR }}/tests.py -vs
+
+about:
+    home: http://github.com/soft-matter/slicerator
+    license: BSD
```
Comment (jakirkham) [inline recipes/slicerator/meta.yaml]: Please add how many clauses.

**Comment by nkeim** [comment]
@ericdill Thanks for letting us know! I would like to be added as a maintainer on the trackpy feedstock once you get around to it. I imagine that @caspervdw, @danielballan, and maybe @tacaswell would like to be maintainers on all three packages, but I leave that up to them.

I take it that in principle this will make many conda packages in the `soft-matter` channel redundant, but that the core contributors will still be responsible for development branch packages between releases (if desired).

Also, as you may know we already have conda recipes in the pims and trackpy repositories. Hopefully that will simplify things.

Also, re: @jakirkham 's request: it is 3 clauses.

**Comment by ericdill** [comment]
> I would like to be added as a maintainer on the trackpy feedstock once you get around to it

Sure thing.

> I take it that in principle this will make many conda packages in the soft-matter channel redundant

Only the tagged releases

> that the core contributors will still be responsible for development branch packages between releases (if desired)

There is [discussion on how to do development releases with conda-forge] (https://github.com/conda-forge/conda-forge.github.io/issues/147) which would likely be quite a bit simpler than building a set of ~15 binaries by hand for 3 flavors of python, 3 platforms and 32/64bit architectures, so perhaps conda-forge could entirely take the place of the `soft-matter` channel, but that is not necessarily my goal here.

**Comment by nkeim** [comment]
This all sounds good. Thanks, @ericdill .

**Comment by ericdill** [comment]
@jakirkham @ocefpaf Could you give this another review? It should be good to go

**Comment by ericdill** [comment]
Thanks!

**Comment by ocefpaf** [comment]
LGTM

**Pull request merged**